README.md

```diff
@@ -16,13 +16,38 @@ Let's get your machine setup to startup the application!
 ⚠️  We assume you already have ruby installed with your preferred version manager. This codebase supports [rbenv](
 https://github.com/rbenv/rbenv) and [asdf](https://github.com/asdf-vm/asdf-ruby).
 
+## 🤝 Contributing Guidelines 
+
+Please feel free to contribute! Priority will be given to pull requests that address outstanding issues and have appropriate test coverage. Focus on issues tagged with the next milestone for higher priority.
+
+To contribute:
+* Identify an unassigned issue
+* Assign the issue to yourself to avoid duplicated efforts (or request assignment by adding a comment)
```

Comment:
I think we can remove the 'assign yourself...' bit as only people added to permissions in the repo can do this and it's not something done often. So best to just comment on an issue and await to be assigned. Also worth stating, only work on one issue at a time.